application-templates/starter/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A starter example for the bare minimum setup to develop a Merchant Center Application",
   "private": true,
   "scripts": {
+    "prebuild": "exit 0",
```

Comment:
A bit weird to ship a starter template with this. Is there another way to avoid doing this?

Reply:
Yes, you were looking at an old commit. I removed it. It's was needed cause we had a `prebuild` in the global `package.json` which called into each sub-package to `prebuild`. Which I think should happen anyway if manypkg invokes `build` in a package or we do via `yarn`.